node/tasks.py

```diff
@@ -26,9 +25,9 @@
 def check_finalization() -> None:
     """Check and add not finalized batches to missed batches."""
     for app_name in list(zconfig.APPS.keys()):
-        not_finalized_batches: dict[str, Any] = zdb.get_not_finalized_batches(app_name)
+        not_finalized_batches: list[dict[str, Any]] = zdb.get_still_sequenced_batches(app_name)
         if not_finalized_batches:
-            zdb.add_missed_batches(app_name=app_name, batches_data=not_finalized_batches)
+            zdb.add_missed_batches(app_name, not_finalized_batches)
```

Comment:
here we should get not finalized batches not still sequenced ones and we should not handle this dispute scenario by adding something to missed batches